tawazi/dag.py

```diff
@@ -621,3 +628,110 @@ def _handle_exception(self, graph: DiGraphEx, fut: "Future[Any]", id_: IdentityH
 
                 else:
                     raise NotImplementedError(f"Unknown behavior name: {self.behavior}")
+
+
+# TODO: change the name of twz_nodes!! should be leaves_nodes
```

Comment:
shouldn't we change that now before merging ?

Reply:
I am going to do another PR to change this. It is rather complicated to change because it requires changing a lot of tests which relies on __call__(... twz_nodes)

Reply:
I was planning on removing twz_nodes from `__call__` and implementing `exlcude_nodes` feature in `DAGExecution` directly in a single PR

Reply:
ok